client/post-editor/editor-title/index.jsx

```diff
@@ -66,6 +66,8 @@ export default React.createClass( {
 		} );
 
 		this.onChange( event );
+
+		event.target.scrollLeft = 0;
```

Comment:
Is this something that should really be part of `FormTextInput`? Feels brittle to be monkeying with a DOM element from a custom element.